draft-ietf-quic-transport.md

```diff
@@ -318,8 +318,8 @@ x (?) = C..D:
 : Indicates that x is optional (and has length of E)
 
 x (E) ...:
-: Indicates that x is repeated zero or more times (and that each instance is
-  length E)
+: Indicates that zero or more occurences of x are present (and that each
```

Comment:
```suggestion
: Indicates that zero or more instances of x are present (and that each
```
Fixes a spelling issue (or regional variation?) and adds parallelism in one shot!